.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-10-15 13:52:14 UTC using RuboCop version 0.93.1.
```

Comment:
This file will conflict nicely with the one from the PR to use the shopify rubocop gem. Otherwise it all looks good

Reply:
I already merged #790 so it doesn't seem to causing any problem.